homeworks/23/instagram-api/src/main/java/com/example/RegisterServlet.java

```diff
@@ -0,0 +1,29 @@
+package com.example;
```

Comment:
The Login and Register servlets are working, and their code is very clean and small, just as the rest of his servlets.